handbook/ops/bizops/customer_ops_data.md

```diff
@@ -0,0 +1,38 @@
+# CustomerOps Data Workflows
```

Comment:
```suggestion
# Data Workflows
```

I think we should remove the customer ops framing of this since we use these for lots of different reasons

Reply:
(And I won't go through and suggest on all the customer ops references but I think the same thing for all of them 😄)